apt/minimization/minimizer.py

```diff
@@ -674,7 +679,10 @@ def _attach_cells_representatives(self, prepared_data, originalTrainFeatures, la
             sample_rows = prepared_data.iloc[indexes]
             sample_labels = labels_df.iloc[indexes]['label'].values.tolist()
             # get rows with matching label
-            indexes = [i for i, label in enumerate(sample_labels) if label == cell['label']]
+            if self.is_regression:
+                indexes = [i for i, label in enumerate(sample_labels)]
```

Comment:
I think there is probably a more concise way of writing this without list comprehension (isn't it just that match_samples = sample_rows and match_rows = original_rows?)